src/AppController.m

```diff
@@ -186,7 +186,6 @@ -(void)awakeFromNib
 	mainWindow.title = self.appName;
 	[NSApplication sharedApplication].delegate = self;
 	mainWindow.minSize = NSMakeSize(MA_Default_Main_Window_Min_Width, MA_Default_Main_Window_Min_Height);
-    [mainWindow setAllowsConcurrentViewDrawing:YES];
```

Comment:
You're right. But it won't harm either.
Changes in `MainMenu.nib` might be more interesting…